libraries/adaptive-expressions/src/builtinFunctions/formatNumber.ts

```diff
@@ -24,36 +24,36 @@ export class FormatNumber extends ExpressionEvaluator {
     }
 
     private static evaluator(): EvaluateExpressionDelegate {
-        return FunctionUtils.applyWithOptionsAndError(
-            (args: any[], options: Options): any => {
-                let value: any = null;
-                let error: string;
-                let number = args[0];
-                let precision = args[1];
-                let locale = options.locale ? options.locale : 'en-us';
-                locale = FunctionUtils.determineLocale(args, locale, 3);
-                if (typeof number !== 'number') {
-                    error = `formatNumber first argument ${number} must be a number`;
-                } else if (typeof precision !== 'number') {
-                    error = `formatNumber second argument ${precision} must be a number`;
-                } else if (locale && typeof locale !== 'string') {
-                    error = `formatNubmer third argument ${locale} is not a valid locale`;
+        return FunctionUtils.applyWithOptionsAndError((args: any[], options: Options): any => {
+            let value: any = null;
+            let error: string;
+            const number = args[0];
+            const precision = args[1];
+            let locale = options.locale ? options.locale : 'en-us';
```

Comment:
I think it might be cleaner to push the default `en-us` to the `determineLocale` method. I.e.

```typescript
public static determineLocale(args: any[], maxArgsLength: number, locale = 'en-us'): string
```

Then we don't have to default `locale` in every call site of `determineLocale`, it's defaulted automatically when the `locale` argument is undefined.

Reply:
changed to `determineLocale(args: any[], maxArgsLength: number, locale = 'en-us')`